docs/docs/reference/release-notes/v4.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: "2022-01-25"
+version: "4.6.0"
+title: "v4.6 Release Notes"
+---
+
+Welcome to `gatsby@4.6.0` release (January 2022 #2)
+
+Key highlights of this release:
+
+- [Performance update: Reuse cache across instances](#reuse-cache-across-instances).
+- [Improve live editing experience for images in markdown files](#improve-live-editing-experience-for-image-in-markdown-files)
+
+## Reuse cache across instances
```

Comment:
```suggestion
## Speeding Up Subsequent Queries
```

We want to use https://en.wikipedia.org/wiki/Title_case with the Chicago Manual of Style